easyinstall.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Does this really require Bash? I believe the old version was universal, and if possible, we'd like to keep that